Lecture_27/Homework_27.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <title>Резюме</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <style>
+    table {
+        background-color: gray;
+    }
+</style>
+</head>
+<body>
+  <script>
+		var buttonsContainer = document.createElement("div");
+		buttonsContainer.classList.add("text-center", "mx-auto");
+		
```

Comment:
this container was forgotten to be added to body
```suggestion
		document.body.appendChild(buttonsContainer)
```